src/sst/core/sstinfo.cc

```diff
@@ -152,10 +152,10 @@ addELI(ElemLoader& loader, const std::string& lib, bool optional)
 static void
 processSSTElementFiles()
 {
-    std::vector<bool> EntryProcessedArray;
-    ElemLoader        loader(g_searchPath);
-
-    std::vector<std::string> potentialLibs = loader.getPotentialElements();
+    std::vector<bool>        EntryProcessedArray;
```

Comment:
I know you didn't write this as part of the PR, but `vector<bool>` eeek

Reply:
I understand this is automatically compiled as a bit field anyway.